third_party/openmetadata/Makefile

```diff
@@ -0,0 +1,9 @@
+deploy-openmetadata:
```

Comment:
please add the default make rule, and allow defining the target namespace as an environment variable